Engine/Graphics/Indoor.cpp

```diff
@@ -1212,7 +1212,9 @@ int IndoorLocation::GetSector(int sX, int sY, int sZ) {
 
     // No face found - outside of level
     if (!NumFoundFaceStore) {
-        logger->Warning("Sector fail");
+        if (engine->config->verbose_logging)
+            logger->Warning("Sector fail");
```

Comment:
Should this really be under verbose flag? Unlike the floorlvl fail, this one is a genuine error that shouldn't happen at all, no?

Reply:
Currently they are both spammy as hell. So until they both fixed I personally prefer guard them behind some flag.
But I can revert this one or both if you wish.

Reply:
Interesting. Can you create an issue with examples so that I could look into them?

And I suggest we leave the sector fail warning as it was.

Reply:
Reverted getsector warning.